src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IArgument.vb

```diff
@@ -821,5 +831,198 @@ BC30272: 'y' is not a parameter of 'Public Sub M2(x As Integer)'.
+        End Sub
+
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact>
+        Public Sub InOutConversionUserDefinedMissingOperator()
+            Dim source = <![CDATA[
+Class C
+    Public Shared Widening Operator CType(ByVal c As C) As Integer
+        Return 0
+    End Operator
+End Class
+
+Class Program
+    Sub M1()
+        Dim x = New C()
+        M2(x)'BIND:"M2(x)"
+    End Sub
+
+    Sub M2(ByRef a As Integer)
+    End Sub
+End Class]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IInvalidExpression (OperationKind.InvalidExpression, Type: System.Void, IsInvalid) (Syntax: 'M2(x)')
+  Children(2):
+      IOperation:  (OperationKind.None) (Syntax: 'M2')
+      ILocalReferenceExpression: x (OperationKind.LocalReferenceExpression, Type: C, IsInvalid) (Syntax: 'x')
+]]>.Value
+
+            Dim expectedDiagnostics = <![CDATA[
+BC33037: Cannot copy the value of 'ByRef' parameter 'a' back to the matching argument because type 'Integer' cannot be converted to type 'C'.
+        M2(x)'BIND:"M2(x)"
+           ~
+]]>.Value
+            VerifyOperationTreeAndDiagnosticsForTest(Of InvocationExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
+        End Sub
+
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact>
+        Public Sub GettingInOutConverionFromVBArgument()
+            Dim source = <![CDATA[
+Class C
+    Public Shared Widening Operator CType(ByVal c As C) As Integer
+        Return 0
+    End Operator
+
+    Public Shared Narrowing Operator CType(ByVal i As Integer) As C
+        Return New C()
+    End Operator
+End Class
+
+Class Program
+    Sub M1()
+        Dim x = New C()
+        M2(x)'BIND:"M2(x)"
+    End Sub
+
+    Sub M2(ByRef a As Integer)
+    End Sub
+End Class]]>.Value
+
+            Dim fileName = "a.vb"
+            Dim syntaxTree = Parse(source, fileName, options:=Nothing)
+
+            Dim compilation = CreateCompilationWithMscorlib45AndVBRuntime({syntaxTree}, DefaultVbReferences.Concat({ValueTupleRef, SystemRuntimeFacadeRef}))
+            Dim result = GetOperationAndSyntaxForTest(Of InvocationExpressionSyntax)(compilation, fileName)
+
+            Dim operatorNodes = compilation.SyntaxTrees(0).GetRoot().DescendantNodes.Where(Function(n As SyntaxNode) As Boolean
+                                                                                           return n.Kind = SyntaxKind.OperatorStatement
+                                                                                        End Function).ToList()
+            Dim semanticModel = compilation.GetSemanticModel(compilation.SyntaxTrees(0))
+            
+            Dim expectedInKind = ConversionKind.Widening Or ConversionKind.UserDefined
+            Dim exptectedInMethod = CType(semanticModel.GetDeclaredSymbolFromSyntaxNode(operatorNodes(0)), IMethodSymbol)
```

Comment:
>CType(semanticModel.GetDeclaredSymbolFromSyntaxNode(operatorNodes(0)), IMethodSymbol) [](start = 36, length = 85)

This looks more complicated than it should be. There is no need to use semantic model to find methods, I believe that could be done by name.